engine/src/main/java/org/hibernate/search/engine/common/spi/SearchIntegrationEnvironment.java

```diff
@@ -68,19 +66,37 @@ private SearchIntegrationEnvironment(Builder builder) {
 			serviceResolver = DefaultServiceResolver.create( aggregatedClassLoader );
 		}
 
+		propertyChecker = builder.propertyChecker;
 		beanProvider = builder.beanProvider;
+
+
+		ConfigurationPropertySource userPropertySource = builder.propertySource.withMask( CONFIGURATION_PROPERTIES_MASK );
+
+		propertySource = userPropertySource.withFallback(
+				ConfigurationPropertySourceScopeUtils.fallback( BeanResolverImpl.create(
+						classResolver, serviceResolver, beanProvider,
+						userPropertySource
```

Comment:
Thanks! 🎉 😃

Reply:
I just force-pushed, there was a syntax error in the javadoc that my local build didn't detect since I didn't use `-Pdist`... sorry... Should be all good now.

Reply:
thanks, no worries 😃